### PR TITLE
Update page file generating 

### PR DESCRIPTION
**Pull request by JuneZhou19**
@xiar @weia5-emc @turtle-fly @XiaowenJiang @chenge3 @chenc52 @fub2

### CODE REVIEW

File: docs/configuration.rst
Comment (fub2) [inline docs/configuration.rst]: page data sounds a little bit general. how about a little bit more elaboration about this is supporting defining/injecting mode sense and inquiry data in any pages.

File: docs/feature.rst
Comment (fub2) [inline docs/feature.rst]: same as above - Data in INQUIRY/MODE SENSE pages

File: docs/how_to.rst
Comment (turtle-fly) [inline docs/how_to.rst]: Add a space to make a different indent, so this line will be displayed as code. @weia5-emc @JuneZhou19

File: docs/how_to.rst
Comment (turtle-fly) [inline docs/how_to.rst]: This line as well~add a space to make a different indent, so this line will be displayed as code. @weia5-emc @JuneZhou19

File: docs/configuration.rst
Comment (turtle-fly) [inline docs/configuration.rst]: `<link>` has not been replaced to real link~ @weia5-emc @JuneZhou19

**Review by turtle-fly — CHANGES_REQUESTED** [review]

File: docs/configuration.rst
Comment (weia5-emc) [reply docs/configuration.rst]: will add more detail information.

File: docs/configuration.rst
Comment (weia5-emc) [inline docs/configuration.rst]: the python script is still in unpublished state.  we will make the link first, then publish the tool.

**Comment by turtle-fly** [comment]
OK to me, up to you @fub2

**Comment by JuneZhou19** [comment]
@turtle-fly @fub2 Conflict resolved.

File: docs/feature.rst
Comment (fub2) [inline docs/feature.rst]: Duplicate

File: docs/user_guide.rst
Comment (fub2) [inline docs/user_guide.rst:65]: I would suggest to move these usage on sg_format and hdparm into how-to section. Reason is it is describing how to exercise drive erasure feature, either infrasim implemented or physical driver native supported. 

Instead we could simple put a statement that sg-format and hdparm are supported to exercise our feature; if there're arguments supporting defining erasure duration, progressing, that content should be placed in user guide section.

Last comment is can we put a bullet as drive emulation (both feature, user guide chapters) and place drive erasure and INQUIRY/MODE SENSE data injection features under this bullet?

@JuneZhou19 @chenc52 thoughts?

Comment (JuneZhou19) [reply docs/user_guide.rst:65]: Sounds make sense. How about to merge this PR if no more question for page file related content? @fub2 
Since PR for drive erase is already merged, we plan to revise for your comments after Julie's PR for time customizing merged.

**Comment by JuneZhou19** [comment]
@fub2 Good catch around feature.rst! Doc Updated.

**Review by turtle-fly — APPROVED** [review]

File: docs/user_guide.rst
Comment (turtle-fly) [reply docs/user_guide.rst:65]: @JuneZhou19 ok to me, please mark this action~

**Pull request merged**
